src/sbt-test/lagom-conductr-bundle/lagom-java-service-execution/project/Build.scala

```diff
@@ -0,0 +1,51 @@
+/* 
+ * Copyright (C) 2016-2017 Lightbend Inc. <https://www.lightbend.com>
+ */
+import java.net.HttpURLConnection
+import java.io.{BufferedReader, InputStreamReader}
+
+import sbt.{IO, File}
+
+// copy/pasted and adapted from
+// https://github.com/lagom/lagom/blob/d9f4df0f56f7c567e88602fa91698b8d8f5de3d9/dev/sbt-plugin/src/sbt-test/sbt-plugin/run-all-javadsl/project/Build.scala#L9
+object DevModeBuild {
+
+  val ConnectTimeout = 10000
+  val ReadTimeout = 10000
+
+  def waitForRequestToContain(uri: String, toContain: String)(totalAttempts:Int=10): Unit = {
+    waitFor[String](
+      makeRequest(uri),
+      _.contains(toContain),
+      actual => s"'$actual' did not contain '$toContain'"
+    )(totalAttempts)
+  }
+
+  def makeRequest(uri: String): String = {
+    var conn: java.net.HttpURLConnection = null
+    try {
+      val url = new java.net.URL(uri)
+      conn = url.openConnection().asInstanceOf[HttpURLConnection]
+      conn.setConnectTimeout(ConnectTimeout)
+      conn.setReadTimeout(ReadTimeout)
+      conn.getResponseCode // we make this call just to block until a response is returned.
+      val br = new BufferedReader(new InputStreamReader((conn.getInputStream())))
+      Stream.continually(br.readLine()).takeWhile(_ != null).mkString("\n").trim()
```

Comment:
I'm on an investigation phase trying to keep the amount of new code to a minimum. I knew that approach worked and tried to avoid getting stuck on managing dependencies for the sbt build, or getting to know Play.ws API. Will happily refactor on a separate PR.